libproxy/proxy.h

```diff
@@ -84,13 +84,21 @@ pxProxyFactory *px_proxy_factory_new(void);
  * previous does not exist. As an example, on Mac OS X you can configure a
  * RTSP streaming proxy. The expected returned configuration would be:
  *   - rtsp://[username:password@]proxy:port
+ *
+ * To free the returned value, call px_proxy_factory_free_proxies.
  * 
  * @url The URL we are trying to reach
  * @return A NULL-terminated array of proxy strings to use
  */
 char **px_proxy_factory_get_proxies(pxProxyFactory *self, const char *url);
 
 /**
+ * Frees the proxy array returned by px_proxy_factory_get_proxies when no
+ * longer used.
+ */
```

Comment:
Please add a "Since: 0.4.16" tag to the comment block - (0.4.16 being the next release)

Reply:
Done.